meta/TreeMasses.m

```diff
@@ -408,6 +409,16 @@
 
 IsElectricallyCharged[par_] := GetElectricCharge[par] != 0;
 
+IsChargedUnder[field_, vector_?IsVector] := 
+  Which[(*Check 2 special cases first which are quicker*)
+    IsPhoton[vector], IsElectricallyCharged[field];,
+    IsGluon[vector], ColorChargedQ[field];,
```

Comment:
Sorry, I think you need to remove the `;` in all three branches, because otherwise the function will not return anything.

Reply:
Yes, thanks for spotting that!

Reply:
Sorry, I think you forgot the semicolon in the last branch. :)

Reply:
Thanks (Note to self, no longer make code alterations first thing after getting up...)!